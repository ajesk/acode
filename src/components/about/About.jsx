import React from 'react';
import './About.scss';
const profile = require('../../resources/me.jpeg')

const About = () => {
  return (
    <div className="about">
        <img src={profile} alt="aaron profile"></img>
        <ul>
          <li>He is a full stack developer with experience ranging from embedded architecture to ui development</li>
          <li>He enjoys cooking</li>
          <li>He enjoys trying craft beers/whiskey/wine</li>
          <li>He recently picked up woodworking as a hobby</li>
          <li>He has a loving fianc√® (to be married soon)</li>
          <li>He has two cats (Vanilla & Crash) and one dog (Jace)</li>
        </ul>
      </div>
  );
}

export default About;
