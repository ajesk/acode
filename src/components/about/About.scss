ul {
  list-style-type: none;
}

li {
  box-shadow: 1px 1px 1px 0 #ccc;
  border: 0.25px solid black;
  padding: 1em;
  margin: 0.2em;
  border-radius: 5px;
  
  button {
    border: none;
    background-color: inherit;
    font-family: inherit;
    font-size: large;
    width: 100%;
    height: 100%;
  }
}


img {
  height: 300px;
  padding: 0.25em
}

.selected {
  box-shadow: 2px 2px 1px 0 #ccc;
  background-color: lightgray;
}

.about {
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: flex-start;
  width: 80%;

  .about-left {
    flex: 0 0 auto;
    margin-right: 2em;
  }

  .about-right {
    flex: 5;
  }

  .about-detail-enter {
    opacity: 0;
  }
  .about-detail-enter-active {
    opacity: 1;
    transition: opacity 500ms ease-in;
  }
  .about-detail-exit {
    opacity: 1;
  }
  .about-detail-exit-active {
    opacity: 0;
    transition: opacity 500ms ease-in;
  }
}

