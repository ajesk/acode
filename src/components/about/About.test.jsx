import React from 'react';
import { render } from '@testing-library/react';
import About from './About';

describe('about', () => {
  test('snapshot', () => {
    expect(render(<About />)).toMatchSnapshot();
  });
});
