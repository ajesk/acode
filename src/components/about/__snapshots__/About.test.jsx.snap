// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`about snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="about"
      >
        <img
          alt="aaron profile"
          src="me.jpeg"
        />
        <ul>
          <li>
            He is a full stack developer with experience ranging from embedded architecture to ui development
          </li>
          <li>
            He enjoys cooking
          </li>
          <li>
            He enjoys trying craft beers/whiskey/wine
          </li>
          <li>
            He recently picked up woodworking as a hobby
          </li>
          <li>
            He has a loving fiancè (to be married soon)
          </li>
          <li>
            He has two cats (Vanilla & Crash) and one dog (Jace)
          </li>
        </ul>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="about"
    >
      <img
        alt="aaron profile"
        src="me.jpeg"
      />
      <ul>
        <li>
          He is a full stack developer with experience ranging from embedded architecture to ui development
        </li>
        <li>
          He enjoys cooking
        </li>
        <li>
          He enjoys trying craft beers/whiskey/wine
        </li>
        <li>
          He recently picked up woodworking as a hobby
        </li>
        <li>
          He has a loving fiancè (to be married soon)
        </li>
        <li>
          He has two cats (Vanilla & Crash) and one dog (Jace)
        </li>
      </ul>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
